python/cudf/cudf/tests/test_index.py

```diff
@@ -2831,7 +2833,7 @@ def test_rangeindex_append_return_rangeindex():
 
 
 index_data = [
-    range(np.random.randint(0, 100)),
+    range(rng.integers(0, 100)),
```

Comment:
TODO: Can we open an issue to turn this usage into a fixture, so that we don't do this at test collection time?

Reply:
Made this change in this PR.